backend/models/distance.py

```diff
@@ -66,11 +77,105 @@ def get_distance_in_miles(self, point_a, point_b):
         return
 
 
+class OSRMDrivingDistance(MeasureDistance):
+    """
+    Class for OSRM driving distance measurements.
+
+    Uses the `table` OSRM API endpoint.
+    """
+
+    def __init__(self, api_url=None):
+        """Initialize the distance class with the OSRM URL."""
+        self.api_url = os.getenv('OSRM_URL')
```

Comment:
At least check if it is passed in or not, or use config maybe?

Reply:
Done.